apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -13,6 +15,13 @@ import AnimationList from './AnimationList';
 import FrameList from './FrameList';
 import ResizablePanes from './ResizablePanes';
 
+/**
+ * @const {string} domain-relative URL to Piskel index.html
+ * In special environment builds, append ?debug flag to get Piskel to load its own debug mode.
+ */
+const PISKEL_PATH = '/blockly/js/piskel/index.html' +
+    (PISKEL_DEVELOPMENT_MODE ? '?debug' : '');
```

Comment:
Went down this path for a bit and it's actually uglier - these replacements are treated as code fragments, so to use a string you have to do `PISKEL_PATH: '"' + piskelPath + '"'`... and the gruntfile has no special knowledge about where the package is deployed either, so on the whole it doesn't feel much better there than here.  I think I'll leave this as-is and revisit it in a near-future PR.

While I'm at it, I should write a utility for 'package-relative' static URLs, so that at least the `blockly` part of this domain-relative URL can go away.